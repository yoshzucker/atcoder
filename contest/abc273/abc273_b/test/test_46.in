144447000000000 12
