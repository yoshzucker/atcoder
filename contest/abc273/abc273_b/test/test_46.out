145000000000000
